v14 deprecation on change listener

```
foundry-esm.js:4621 Error: You are accessing globalThis.duplicate which must now be accessed via foundry.utils.duplicate
Deprecated since Version 12
Backwards-compatible support will be removed in Version 14
    at Object.logCompatibilityWarning (foundry-esm.js:4609:19)
    at get (foundry-esm.js:66671:25)
    at SWNActorSheet._onResourceChange (actor-sheet.mjs:827:26)
```


Coming from:

`    const resourceList = duplicate(this.actor.system.tweak.resourceList);`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -34,6 +34,7 @@ Verified against Foundry v14.365. Minimum supported core remains 13.345.
 - Fixed the ship sensor roll never posting to chat
 - Fixed ship weapon migration silently aborting on items predating the trauma field
 - Fixed vehicle sheet item descriptions throwing on every expand click
+- Fixed editing character resources, custom currencies, and vehicle cargo throwing on v14 (the `duplicate` global was removed, not just deprecated)
 - Replaced deprecated globals removed in v15, the `renderChatMessage` hook, and the roll mode APIs deprecated in v14 (`TextEditor`/`DragDrop` thanks @illyja)
 - System now loads with no deprecation warnings or errors on v14
 - Contributor docs (`CLAUDE.md`) now cover the v13/v14 rules, with `AGENTS.md` and `GEMINI.md` pointing at it

--- a/module/sheets/actor-sheet.mjs
+++ b/module/sheets/actor-sheet.mjs
@@ -1167,7 +1167,7 @@ export class SWNActorSheet extends SWNBaseSheet {
     event.preventDefault();
     const dataset = target.dataset;
     const idx = dataset.rlIdx;
-    const resourceList = duplicate(this.actor.system.tweak.resourceList);
+    const resourceList = foundry.utils.duplicate(this.actor.system.tweak.resourceList);
     resourceList.splice(idx, 1);
     await this.actor.update({ "system.tweak.resourceList": resourceList });
   }
@@ -1177,7 +1177,7 @@ export class SWNActorSheet extends SWNBaseSheet {
     const value = event.target?.value;
     const resourceType = $(event.currentTarget).data("rlType");
     const idx = $(event.currentTarget).parents(".item").data("rlIdx");
-    const resourceList = duplicate(this.actor.system.tweak.resourceList);
+    const resourceList = foundry.utils.duplicate(this.actor.system.tweak.resourceList);
     resourceList[idx][resourceType] = value;
     await this.actor.update({ "system.tweak.resourceList": resourceList });
   }
@@ -1272,7 +1272,7 @@ export class SWNActorSheet extends SWNBaseSheet {
               value: currencyValue,
               carried: carried,
             };
-            const currencyList = duplicate(this.actor.system.credits.extraCurrencies);
+            const currencyList = foundry.utils.duplicate(this.actor.system.credits.extraCurrencies);
             currencyList[currencyIdx] = currency;
             await this.actor.update({
               "system.credits.extraCurrencies": currencyList
@@ -1298,7 +1298,7 @@ export class SWNActorSheet extends SWNBaseSheet {
               ui.notifications.info("Currency deletion cancelled");
               return "cancel";
             }
-            const currencyList = duplicate(this.actor.system.credits.extraCurrencies);
+            const currencyList = foundry.utils.duplicate(this.actor.system.credits.extraCurrencies);
             currencyList.splice(currencyIdx, 1);
             await this.actor.update({
               "system.credits.extraCurrencies": currencyList
@@ -1310,7 +1310,7 @@ export class SWNActorSheet extends SWNBaseSheet {
           action: "convert",
           icon: 'fas fa-exchange-alt',
           callback: async (_event, _button, _dialog) => {
-            const currencyList = duplicate(this.actor.system.credits.extraCurrencies);
+            const currencyList = foundry.utils.duplicate(this.actor.system.credits.extraCurrencies);
             const baseCurrencyName = game.settings.get("swnr", "baseCurrencyName");
             const currencyToConvert = currencyList[currencyIdx];
             let baseCurrencyToAdd = 0;

--- a/module/sheets/base-sheet.mjs
+++ b/module/sheets/base-sheet.mjs
@@ -659,7 +659,7 @@ export class SWNBaseSheet extends api.HandlebarsApplicationMixin(
 
           if (currencyType === 'custom') {
             const currencyIdx = target.dataset.currencyIdx;
-            let extraCurrencies = duplicate(this.actor.system.credits.extraCurrencies);
+            let extraCurrencies = foundry.utils.duplicate(this.actor.system.credits.extraCurrencies);
             const currency = extraCurrencies[currencyIdx];
             if (currency == undefined || currency == null) {
               ui.notifications?.error("Invalid currency");

--- a/module/sheets/vehicle-sheet.mjs
+++ b/module/sheets/vehicle-sheet.mjs
@@ -452,7 +452,7 @@ export class SWNVehicleSheet extends SWNBaseSheet {
     const value = event.target?.value;
     const resourceType = event.target.dataset.rlType
     const idx = event.target.dataset.rlIdx;
-    const resourceList = duplicate(this.actor.system.cargoCarried);
+    const resourceList = foundry.utils.duplicate(this.actor.system.cargoCarried);
     resourceList[idx][resourceType] = value;
     await this.actor.update({ "system.cargoCarried": resourceList });
   }
@@ -461,7 +461,7 @@ export class SWNVehicleSheet extends SWNBaseSheet {
     event.preventDefault();
     event.stopPropagation();
     const idx = target.dataset.idx;
-    const resourceList = duplicate(this.actor.system.cargoCarried);
+    const resourceList = foundry.utils.duplicate(this.actor.system.cargoCarried);
     resourceList.splice(idx, 1);
     await this.actor.update({ "system.cargoCarried": resourceList });
   }

--- a/templates/chat/stat-block.hbs
+++ b/templates/chat/stat-block.hbs
@@ -26,7 +26,7 @@
 </table>
 <div class="statApplyButton">
   <button data-actor-id="{{actor.id}}">
-    {{formatMessage (localize 'swnr.chat.statApply') name=actor.name}}
+    {{localize 'swnr.chat.statApply' name=actor.name}}
   </button>
 </div>
 {{!--<pre>{{stringify @root}}</pre> --}}

